### ISSUE TITLE
[suggestion] Lit is missing Alpaca_Eval and MT benchmark

### ISSUE THREAD

**Issue by sanyalsunny111** [Lightning-AI/lit-gpt#708]
Lit-gpt has a superb libraries for instruction tuning. But instruction tuned models are popularly never used for eval using lm eval harness. 
MT benchmark: https://lmsys.org/blog/2023-06-22-leaderboard/
alpaca eval: https://github.com/tatsu-lab/alpaca_eval

**Comment by carmocca** [comment]
Is your suggestion to have a tutorial on how to submit lit-gpt models to those leaderboards?

**Comment by sanyalsunny111** [comment]
Yeah that should work. But I am suggesting more in the lines of making lit version of both of these benchmarks. Like you guys did with lm eval harness.
